lstchain/reco/reconstructorCC.py

```diff
@@ -227,6 +227,69 @@ def template_interpolation(gain, times, t0, dt, a_hg, a_lg):
     return out
 
 
+@njit(cache=True)
+def template_interpolation_withNSB(empty_waveform, time, is_high_gain, additional_nsb, amplitude, t_0,
```

Comment:
The method name is not really representative of what this code actually does. It is not the template interpolation method with NSB. This method creates NSB only waveforms.

Reply:
Thanks, @gabemery, for your review! I like `nsb_only_waveforms`. What do you think?

Reply:
I think it works